ecommerce_integrations/unicommerce/custom/delivery_note.json

```diff
@@ -0,0 +1,420 @@
+{
+ "custom_fields": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 1,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2023-04-07 21:02:05.946548",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Delivery Note",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "unicommerce_details",
+   "fieldtype": "Section Break",
+   "hidden": 0,
+   "hide_border": 0,
+   "hide_days": 0,
+   "hide_seconds": 0,
+   "idx": 157,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_preview": 0,
+   "in_standard_filter": 0,
+   "insert_after": "instructions",
+   "is_system_generated": 0,
+   "is_virtual": 0,
+   "label": "Unicommerce Details",
+   "length": 0,
+   "mandatory_depends_on": null,
+   "modified": "2023-04-07 21:02:05.946548",
+   "modified_by": "Administrator",
+   "module": null,
+   "name": "Delivery Note-unicommerce_details",
+   "no_copy": 0,
+   "non_negative": 0,
+   "options": null,
+   "owner": "Administrator",
+   "permlevel": 0,
+   "precision": "",
+   "print_hide": 0,
+   "print_hide_if_no_value": 0,
+   "print_width": null,
+   "read_only": 0,
+   "read_only_depends_on": null,
+   "report_hide": 0,
+   "reqd": 0,
+   "search_index": 0,
+   "translatable": 0,
+   "unique": 0,
+   "width": null
+  },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2023-04-07 21:02:25.665457",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Delivery Note",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "unicommerce_order_no",
```

Comment:
Custom fields should be managed by unicommerce setting's `setup_custom_fields` function. 


Fixutres are auto imported which shouldn't be the case for integrations which are not enabled by default.

Reply:
Check this

https://github.com/frappe/ecommerce_integrations/blob/3cf612bd468cec39fc7fe722c3826226245002cf/ecommerce_integrations/unicommerce/doctype/unicommerce_settings/unicommerce_settings.py#L182

Reply:
Added